src/main/java/org/apache/commons/compress/utils/ByteUtils.java

```diff
@@ -266,4 +270,24 @@ private static void checkReadLength(final int length) {
             throw new IllegalArgumentException("Can't read more than eight bytes into a long value");
         }
     }
+
```

Comment:
It looks like this method is only used from the `sevenz` package so let's keep it package-private there and avoid increasing the public API surface.

Reply:
Right, I move it on `AES256SHA256Decoder` because it is used to convert password from char to byte for encryption/decryption